pjsip/include/pjsua-lib/pjsua.h

```diff
@@ -1196,6 +1196,19 @@ typedef struct pjsua_callback
                               pjsip_transaction *tsx,
                               pjsip_event *e);
 
+    /**
+     * This is a general notification callback which is called whenever
+     * a transaction within the account has changed state. Application can
+     * implement this callback for example to monitor the state of
+     * outgoing requests, or to answer unhandled incoming requests
+     * with a final response.
+     *
+     * @param acc_id   Account identification.
+     * @param e         Transaction event that caused the state change.
+     */
+    void (*on_acc_tsx_state)(pjsua_acc_id acc_id,
+                             pjsip_event *event);
+
```

Comment:
Additional param:
- user data/token.

Also for consistency, perhaps the name should be something like `on_acc_send_request` instead of `on_tsx_state`, as the callback is actually called once, i.e: on transaction termination, instead of every tsx state shift.